tests/integration/targets/iptables_state/aliases

```diff
@@ -1,5 +1,5 @@
 destructive
-shippable/posix/group4
```

Comment:
Please stop changing this all the time. You're mainly wasting CI resources.

Reply:
yes, sorry, this was to trigger a new CI build since the checks randomly failed due to another module (etcd3). I thought nobody would review my PR if CI check was not in success first. It seems I was wrong :)